dwave/cloud/client/base.py

```diff
@@ -609,12 +639,22 @@ def _shutdown_threads(self, wait: bool = True):
                                 self._poll_workers, self._load_workers):
                 worker.join()
 
-    def close(self):
+    def close(self, wait: bool = DEFAULT_WAIT_ON_CLOSE):
```

Comment:
If we change `Client.DEFAULT_WAIT_ON_CLOSE` it won't change the behavior of this method. So we should either do `Client._DEFAULT_WAIT_ON_CLOSE` or make this a `None` etc etc.